rhwp/rhwp-studio/src/ui/agent-sidebar/agent-sidebar.css

```
@@ -966,6 +997,91 @@ body.ag-sidebar-resizing .ag-resize-handle::after {
+  color: var(--ag-accent-strong);
+}
+.ag-skill-icon-option:has(input:focus-visible) {
+  outline: 2px solid var(--focus-ring);
+  outline-offset: -3px;
+}
+.ag-skill-icon-option input {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)